src/modules/navigator/follow_target.cpp

```diff
@@ -400,3 +400,37 @@ bool FollowTarget::target_position_valid()
 	// need at least 1 continuous data points for position estimate
 	return (_target_updates >= 1);
 }
+
+void
+FollowTarget::set_follow_target_item(struct mission_item_s *item, float min_clearance, follow_target_s &target,
```

Comment:
just wondering. how tested is the "following target" feature?

Reply:
It was well tested when it first came in. I'm not aware of any testing it's had since. We should get something in place before we break it too badly.